mne/viz/epochs.py

```diff
@@ -844,6 +844,9 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
     # Draw event lines for the first time.
     _plot_vert_lines(params)
 
+    # default key to close window
+    params['close_key'] = 'escape'
```

Comment:
And you probably want to set this also in ``_plot_onkey`` and other key handlers. There are at least the handler for annotation dialog and selection dialog.